docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.1"
+
+services:
+  db:
+    image: mysql:5.6
+    volumes:
+      - ./docker/ecco-my.cnf:/etc/mysql/conf.d/ecco-my.cnf
+    ports:
+      - 3306:3306
+    command: --default-authentication-plugin=mysql_native_password
```

Comment:
Hm, this should already be the default. I will try and drop it on Monday and try again. I used https://docs.docker.com/samples/library/mysql/#-via-docker-stack-deploy-or-docker-compose as a starting point.

Reply:
https://dev.mysql.com/doc/refman/5.6/en/authentication-plugins.html states it's the default. For 5.7 and onwards that statement is gone, probably why it was explicit in the example.

Reply:
402b2ae